app/src/main/kotlin/com/bitwarden/authenticator/ui/platform/feature/settings/SettingsScreen.kt

```diff
@@ -150,6 +169,12 @@ fun SettingsScreen(
                         viewModel.trySendAction(SettingsAction.DataClick.BackupClick)
                     }
                 },
+                onSyncWithBitwardenClick = remember(viewModel) {
+                    {
+                        viewModel.trySendAction(SettingsAction.DataClick.SyncWithBitwardenClick)
```

Comment:
Can we get a test for this click event?

Reply:
Yep! I added tests for handling each of the new events as well.